internal/manifests/manifests.go

```diff
@@ -37,26 +39,32 @@ func BuildAll(params Params) ([]client.Object, error) {
 	configMaps, err := config.BuildConfigMap(params.Tempo, config.Params{S3: config.S3{
 		Endpoint: params.StorageParams.S3.Endpoint,
 		Bucket:   params.StorageParams.S3.Bucket,
-	}})
+	}, HTTPEncryption: params.Gates.HTTPEncryption,
+		GRPCEncryption: params.Gates.GRPCEncryption})
 	if err != nil {
 		return nil, err
 	}
 
-	ingesterObjs, err := ingester.BuildIngester(params.Tempo)
+	ingesterObjs, err := ingester.BuildIngester(params.Tempo, params.Gates)
```

Comment:
In #158 I had to pass down the tempo configuration file checksum to each pod, so I followed Loki's approach to pass it down in the `param` struct. I think we could also add the `Gates` field to the `params` struct, and pass it down to all `Build*` functions instead of passing each var separately, wdyt?

Reply:
yes, I think it makes more sense, I'll change this to pass all the params.